tic_tac_toe.py

```diff
@@ -1,5 +1,7 @@
 from src.game import Game
 
 if __name__ == "__main__":
+    board = Game.board
     game = Game()
     game.display_welcome_message()
+    game.display_board(board)
```

Comment:
This file should represent the entrance to the program and kick things off.  It should have some configuration, and then. start things off.  In its current state, it appears that the game logic itself has leaked into it. I think you got to the point where things "work" but they are not yet "right".  One way to drive this forward is to write the code we expect to work.  So I suggest this code change, but it should force you to to make changes in other parts of the code.

```suggestion
    game = Game()
    game.run()
```